### ISSUE TITLE
Enabling video_recorder_plugin in Windows causes exception (Trac #31)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#35]
Enabling video_recorder_plugin in Windows causes the following exception (regardless of whether or not the webcam is plugged in):

  [Errno 9] Bad file descriptor

Migrated from http://microfluidics.utoronto.ca/ticket/31
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Enabling video_recorder_plugin in Windows causes the following exception (regardless of whether or not the webcam is plugged in):\n\n  [Errno 9] Bad file descriptor", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Enabling video_recorder_plugin in Windows causes exception", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T23:24:05", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120105T03:28:16: christian@fobel.net commented:

> commit f1195320015d9ff62757ed0d377e98865684f737
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Jan 4 22:18:28 2012 -0500
> 
>     [SUBMODULE] Update opencv module, See #35

**Comment by cfobel** [comment]
Trac update at 20120105T19:21:09: 
* cfobel commented:

> Replying to [ticket:31 cfobel]:
> > Enabling video_recorder_plugin in Windows causes the following exception (regardless of whether or not the webcam is plugged in):
> > 
> >   [Errno 9] Bad file descriptor
> 
> 
> This exception doesn't seem to be happening anymore.  However, within the video_recorder_plugin, the following exception occurs in Windows when wrapping the Recorder() initialization in a 'with Silence(...)' statement, attempting to redirect stdout and stderr to a StringIO object:
> 
> ```
> [Error 32] The process cannot access the file because it is being used by another process: u'c:\\users\\christ~1\\appdata\\local\\temp\\...'
> ```
> 


* cfobel changed status from "new" to "accepted"

**Comment by cfobel** [comment]
Trac update at 20120105T19:47:10: 
* christian@fobel.net commented:

> commit fd9572f3d4729acb3e9708ca2e4f7021abe14e1e
> Author: Christian Fobel <christian@fobel.net>
> Date:   Thu Jan 5 14:38:10 2012 -0500
> 
>     [SUBMODULE] Update opencv module.  Fixes #35.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "accepted" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
